app-typescript/components/avatar/avatar-placeholder.tsx

```diff
@@ -20,15 +21,35 @@ export interface IPropsAvatarPlaceholder {
 
 export class AvatarPlaceholder extends React.PureComponent<IPropsAvatarPlaceholder> {
     render() {
-        const {size, tooltip, icon} = this.props;
+        const {size, tooltip, icon, kind, onClick} = this.props;
 
         return (
             <AvatarWrapper
                 size={size}
                 isEmpty={false}
                 icon={icon}
             >
-                <AvatarContentAdd tooltipText={tooltip ?? undefined} onClick={this.props.onClick} />
+                {(() => {
+                    if (kind === 'plus-button') {
+                        return (
+                            <AvatarContentAdd
+                                tooltipText={tooltip ?? undefined}
+                                onClick={onClick}
+                            />
+                        );
+                    } else if (kind === 'user-icon') {
+                        return (
+                            <span onClick={onClick}>
```

Comment:
Interactive/clickable elements should probably be a `button` instead of a `span` (@fritzSF your thoughts?).

As we're adding click support to an `AvatarContentImage`, it would make sense to add this support into that component directly rather than for this single use case (just like we're already doing with the `AvatarContentAdd` component).

Reply:
I've tried a button, but it was breaking styles, so I've used `role="button"` instead.